crates/mysten-common/src/backoff.rs

```diff
@@ -34,31 +33,23 @@ use rand::Rng as _;
 /// ```
 #[derive(Debug, Clone)]
 pub struct ExponentialBackoff {
-    current: Duration,
+    next: Duration,
     factor: f64,
     max_delay: Duration,
     max_jitter: Duration,
 }
 
 impl ExponentialBackoff {
     /// Constructs a new exponential backoff generator, specifying the maximum delay.
-    pub fn new(max_delay: Duration) -> ExponentialBackoff {
+    pub fn new(initial_delay: Duration, max_delay: Duration) -> ExponentialBackoff {
         ExponentialBackoff {
-            current: Duration::from_millis(50),
-            factor: 1.5,
+            next: initial_delay,
+            factor: 1.2,
```

Comment:
Will change this to `1.2` affect other components using this? 

Also please update the comments on the `factor` method as well as it mentions default `1.5` . Maybe also extract to a const?